src/macaron/slsa_analyzer/checks/provenance_available_check.py

```diff
@@ -83,414 +58,6 @@ def __init__(self) -> None:
         ]
         super().__init__(check_id=check_id, description=description, depends_on=depends_on, eval_reqs=eval_reqs)
 
-    def find_provenance_assets_on_package_registries(
```

Comment:
Am I correct to say that this method is completely removed because we have functions that obtain provenances from NPM and Jfrog here?
- NPM - https://github.com/oracle/macaron/blob/927d8c3a2e59b0b323036ff5bd309fa765248899/src/macaron/repo_finder/provenance_finder.py#L143
- Jfrog - https://github.com/oracle/macaron/blob/927d8c3a2e59b0b323036ff5bd309fa765248899/src/macaron/repo_finder/provenance_finder.py#L282

Reply:
Yes.